BUILD.gn

```diff
@@ -594,6 +595,9 @@ if (is_mac) {
       "//ui/strings",
       "//content:sandbox_helper_win",
     ]
+    if (!is_component_build && is_component_ffmpeg) {
+      configs += [ "//build/config/gcc:rpath_for_built_shared_libraries" ]
```

Comment:
would it better to add a `is_linux` check here ?

Reply:
What would be the reasoning for that?

My reasoning for choosing this combination of features is that it corresponds directly with the cause of the issue: building ffmpeg as a shared library when everything else is expecting not to have to load any shared libraries. It's sort of a coincidence that the problem doesn't occur on macOS or Linux.

Reply:
> It's sort of a coincidence that the problem doesn't occur on macOS or windows.

Thats exactly the reason I would want to add that check, there isn't any config changes but it would be clear that the problem is fixed on linux the current way.

Reply:
Hmm, I see your point. I went to look at what this actually ends up doing on macOS and Windows, and it allows putting shared libs in a separate folder using the `gcc_target_rpath` build arg. I'm not sure we'd ever want that, though.

I'll put this in an `is_linux` section, but there's also part of me that wants to keep the build flags across platforms as close as possible.

Reply:
> but there's also part of me that wants to keep the build flags across platforms as close as possible.

Definitely agree on that, but I don't think we will end up using this particular config on other platforms.